README.md

```diff
@@ -1,76 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# practice-tdd
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> This app was built with the purpose to practice Test Driven Development.
 
```

Comment:
- [ ] For the project description, kindly explain what the project is about and what it does, instead of why it was built. This will make your readme to be more descriptive to users.